kolibri/content/constants/presets.py

```diff
@@ -0,0 +1,29 @@
+"""
+This module contains constants representing the ids of FormatPreset.
+"""
+from django.utils.translation import ugettext_lazy as _
+
+# constants for FormatPreset
+HIGH_RES_VIDEO = "high_res_video"
+MID_RES_VIDEO = "mid_res_video"
+LOW_RES_VIDEO = "low_res_video"
+VECTOR_VIDEO = "vertor_video"
+
+EN_SUBTITLE = "en_subtitle"
+ES_SUBTITLE = "es_subtitle"
+ZH_SUBTITLE = "zh_subtitle"
```

Comment:
I would like to *stress* the need to distinguish between the **transcript** of the video dialog in its original language, and subsequent **translations** of that transcript. Granted I'm still trying to grasp how the above code translates to user experience, but if we are talking of loading/adding the available "subtitles" for the chosen video content, I think it's important that user can specify if they are choosing to upload the **original transcript** or **translated transcript** (combined with language?). Maybe this is not the appropriate place for this distinction, but `"es_subtitle"` just does not give me the idea if that's transcript of video originally in Spanish, or Spanish translation of the original transcript in English or other language.

I'd also pepper this a bit more and vote we leave the TV era term *subtitle* behind, and use *transcript* as more appropriate, or even jump ahead the a11y-curve and adopt the term **captions** directly. It's true that captions usually contain more information than most of our transcripts, but we can dream a better world for the future... ;)